pkg/storage/secret/metadata/keeper_store.go

```diff
@@ -285,6 +283,7 @@ func (s *keeperMetadataStorage) List(ctx context.Context, namespace xkube.Namesp
 	if err != nil {
 		return nil, fmt.Errorf("listing keepers %q: %w", q, err)
 	}
+	defer func() { _ = rows.Close() }()
```

Comment:
This was missing from the previous PR. Adding to make it simpler. also doing some formating in the file

Reply:
Nice catch